Makefile-libdnf.am

```diff
@@ -20,7 +20,7 @@
 # Recurse into libdnf
 ALL_LOCAL_HOOKS += libdnf-local
 libdnf-local:
-	cd libdnf-build && $(MAKE)
+	cd libdnf-build && $(MAKE) $(if $(subst 0,,$(V)),VERBOSE=1,)
```

Comment:
Hi, I am not too experienced with Makefile syntax. But what I am understanding from basic searching is that, we are trying to substitute 0 from `$(V)` into empty characters, and if that succeeds, We do $(MAKE) VERBOSE=1?

I have a confusion about what is $(V) tho. But seeing from the commit messages, this line seems to be helping the debugging, should i just "merge"(rh-atomic-bot r+) it then? :D. Let me know.

Reply:
`$(V)` is defined by Automake - you have to look at the generated `Makefile` to understand it.  But honestly I wouldn't spend too much time doing that...for this PR I'd just trust me :smile: - but it's still useful to have "this looks not totally broken" level of review.